DecipherNow/gm-fabric-dashboard
# chore: Adjust cards to equal width in Edge

Resolves #1140

## Proposed Changes

  - Add "overflow: hidden" to TabLink to ensure the cards are equal width in Edge 15/16
  
## How to Test

  - View the instance dashboard cards in all supported browsers.
  - Validate that the cards are equal width in Edge 15/16.
  - Ensure that there aren't any regressions in the other browsers.
